Winget-AutoUpdate/functions/Test-ListPath.ps1

```diff
@@ -0,0 +1,67 @@
+#Function to check Black/White List External Path
+
+function Test-ListPath ($ListPath, $UseWhiteList, $WingetUpdatePath) {
+    # UNC, Web or Local Path
+    if ($UseWhiteList){
+        $ListType="included"
+    }
+    else {
+        $ListType="excluded"
```

Comment:
could be `$ListType="excluded_apps.txt"` directly, no ?

Reply:
Yes, it could :)